cmd/tk/init.go

```diff
@@ -82,11 +87,11 @@ func installK8sLib() error {
 	}
 
 	var initialPackages = []string{
-		"github.com/ksonnet/ksonnet-lib/ksonnet.beta.4",
+		"github.com/jsonnet-libs/k8s-alpha/" + version,
 		"github.com/grafana/jsonnet-libs/ksonnet-util",
```

Comment:
so this becomes our remaining ksonnet reference? Can we nuke that one too somehow?

Reply:
`ksonnet-util` is incredibly useful as a compatibility layer as many libraries depend on how `ksonnet-lib` used to work but `k8s-alpha` broke compat for good reason.

Reply:
Yeah, my comment here was entirely about the name not the functionality, and it was more of a moan - I didn't honestly expect you to take me seriously!